lib/availability_resolver.js

```diff
@@ -62,22 +62,33 @@ class AvailabilityResolver {
             if (barcode_array.length === 3 && barcode_array[1] === 'barcode') {
               let recapAvailabilityStatus = barcodesAndAvailability[barcode_array[barcode_array.length - 1]]
 
-              if (recapAvailabilityStatus !== "Item Barcode doesn't exist in SCSB database.") {
+              // If it's not in ReCAP, it's not requestable:
```

Comment:
We should talk with HTC about making "Item Barcode doesn't exist in SCSB database." a little less of a magic string.